templates/internet-of-things/appstore.html

```diff
@@ -0,0 +1,228 @@
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block" id="ubuntu-com_appstore_left">
+      <!-- rtp section start -->
+      <div class='p-heading-icon u-no-margin--bottom'>
+        <div class='p-heading-icon__header u-no-margin--bottom'>
+          <img class='p-heading-icon__img' src='{{ ASSET_SERVER_URL }}cd7a3a3c-white-paper-icon.svg' alt='' style='max-width: 30px;' />
+          <h3 class='p-heading-icon__title p-muted-heading' style='color: #666;'>White paper</h3>
+        </div>
+      </div>
+      <h2 class='p-heading--four'>
+        <a class='p-link--external' href='https://pages.ubuntu.com/IOT_IoTReport2017.html' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'appstore - left', 'eventLabel' : 'White paper - How to pick a winning IoT business model', 'eventValue' : '1' });'><span hidden>Read the white paper </span>How to pick a winning IoT business model</a>
+      </h2>
+      <!-- rtp section end -->
+    </div>
+    <div class="col-4 p-divider__block" id="ubuntu-com_appstore_center">
+      <!-- rtp section start -->
+      <div class='p-heading-icon u-no-margin--bottom'>
+        <div class='p-heading-icon__header u-no-margin--bottom'>
+          <img class='p-heading-icon__img' src='{{ ASSET_SERVER_URL }}26797d10-newsfeed-stripvideo-icon.svg' alt='' style='max-width: 30px;' />
+          <h3 class='p-heading-icon__title p-muted-heading' style='color: #666;'>Webinar</h3>
+        </div>
+      </div>
+      <h2 class='p-heading--four'>
+        <a class='p-link--external' href='https://www.brighttalk.com/webcast/6793/290087' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'appstore - center', 'eventLabel' : 'Webinar - IoT App Stores: The Path to Post-Sale Revenue', 'eventValue' : '1' });'><span hidden>Watch the webinar </span>IoT App Stores: The Path to Post-Sale Revenue</a>
+      </h2>
+      <!-- rtp section end -->
+    </div>
+    <div class="col-4 p-divider__block" id="ubuntu-com_appstore_right">
+      <!-- rtp section start -->
+      <div class='p-heading-icon u-no-margin--bottom'>
+        <div class='p-heading-icon__header u-no-margin--bottom'>
+          <img class='p-heading-icon__img' src='{{ ASSET_SERVER_URL }}5cf7bd32-news-feed-strip-case-study.svg' alt='' style='max-width: 30px;' />
+          <h3 class='p-heading-icon__title p-muted-heading' style='color: #666;'>Case study</h3>
+        </div>
+      </div>
+      <h2 class='p-heading--four'>
+        <a class='p-link--external' href='https://insights.ubuntu.com/2017/12/11/rocket-chat-communication-platform-enables-simplicity-through-snaps/' onclick='dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'product page', 'eventAction' : 'appstore - right', 'eventLabel' : 'Case study - Rocket.chat archives simplicity through snaps', 'eventValue' : '1' });'><span hidden>Read the case study </span>Rocket.chat archives simplicity through snaps</a>
```

Comment:
This caught my eye: archives -> achieves

Reply:
good catch, but it is being removed... sorry